geomstats/geometry/lie_algebra.py

```diff
@@ -135,17 +136,66 @@ def matrix_representation(self, basis_representation):
 
         return gs.einsum("...i,ijk ->...jk", basis_representation, self.basis)
 
+    def reshape_metric_matrix(self, metric_matrix):
```

Comment:
Would this be better as a function within the module `matrices.py` ?

Reply:
It needs access to `self.matrix_representation` and `self.basis`

Reply:
So both should go there? But the MatrixLieAlgbera class doesn't inherit the Matrices class